docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -33,6 +33,7 @@ This article walks you through the process of enabling single sign-on (SSO) in a
     > * The **Before** folder is a starter project. The UI and other aspects of the add-in that are not directly connected to SSO or authorization are already done. Later sections of this article walk you through the process of completing it. 
     > * The **Completed** version of the sample is just like the add-in that you would have if you completed the procedures of this article, except that the completed project has code comments that would be redundant with the text of this article. To use the completed version, just follow the instructions in this article, but replace "Before" with "Completed" and skip the sections **Code the client side** and **Code the server** side.
     > * The **Completed Multitenant** version is a completed sample that supports multitenancy. Explore this sample if you intend to support Microsoft accounts from different domains with SSO.
+    > *Regardless of which version you use, you will need to trust a cert for the localhost. See the "IMPORTANT" note in the Readme of the repo.*
```

Comment:
Is this meant to be another bullet point or an extension of the 'Completed Multitenant' item? Current behavior is that it's an italicized sentence in the 'Completed Multitenant' bullet point.

Reply:
It's meant to be a completely separate paragraph _after_ the bulleted list. Fixed now.